components/brave_wallet_ui/page/container.tsx

```diff
@@ -365,10 +365,24 @@ function Container (props: Props) {
     }
   }
 
-  const onConnectHardwareWallet = (opts: HardwareWalletConnectOpts): Result.Type<HardwareWalletAccount[]> => {
-    // TODO (DOUGLAS): Add logic to connect a hardware wallet
+  const onConnectHardwareWallet = (opts: HardwareWalletConnectOpts): Promise<HardwareWalletAccount[]> => {
```

Comment:
seems like there should be a test for this

Reply:
I moved this function to another place and split it into 2 different ones, let's leave it as it is for this one, in the next PR the new functions will have tests